src/fixed.rs

```diff
@@ -22,7 +20,7 @@ pub trait FixedInt: Sized + Copy {
         v
     }
     /// Helper: Decode the value from the Vec.
-    fn decode_fixed_vec(v: &Vec<u8>) -> Self {
+    fn decode_fixed_vec(v: &[u8]) -> Self {
```

Comment:
please undo this and similar changes, as the Vec is intended (for backwards compatibility at least, I acknowledge the limited usefulness.

If you disagree very strongly, just delete the whole method, as `decode_fixed` already does this.

Reply:
If you need backwards compatibility, I can revert this and mark this function as deprecated. Any thoughts?